docs/admin-manual/ap-policy-configuration.rst

```diff
@@ -114,7 +114,7 @@ terminates further evaluation of other submit requirements, and is the
 only requirement reported. Each submit requirement is evaluated in the
 context of the *condor_schedd* ClassAd, which is the ``MY.`` name space
 and the job ClassAd, which is the ``TARGET.`` name space. Note that
-``JobUniverse`` and ``RequestMemory`` are both job ClassAd attributes.
+``JobUniverse`` and :ad-attr:`RequestMemory` are both job ClassAd attributes.
```

Comment:
```suggestion
:ad-attr:`JobUniverse` and :ad-attr:`RequestMemory` are both job ClassAd attributes.
```

Reply:
Note that `JobUniverse` is not :classad-attribute-defined: in classad-attributes/job-classad-attributes.rst.

Reply:
And fixed...